src/common/error/src/lib.rs

```diff
@@ -19,7 +19,7 @@ pub mod format;
 pub mod mock;
 pub mod status_code;
 
-pub const GREPTIME_ERROR_CODE: &str = "x-greptime-err-code";
-pub const GREPTIME_ERROR_MSG: &str = "x-greptime-err-msg";
+pub const GREPTIME_DB_HEADER_ERROR_CODE: &str = "X-GreptimeDB-Error-Code";
```

Comment:
These fields don't seem to be referred outside. We'd better unify headers as "X-GreptimeDB-YYY` or `X-Greptime-YYY`. HTTP headers are case-insensitive.

Reply:
To keep b/w compat as much as possible and workaround `HeaderName::from_static` doesn't auto lower inputs, I use `x-greptime-yyy` everywhere.